### PR TITLE
Added haproxy and keepalived config

### PR DESCRIPTION
**Pull request by smanjara**

### CODE REVIEW

File: teuthology/task/ceph_ansible.py [hunk 1]
```diff
@@ -448,6 +451,82 @@ def run_rh_playbook(self):
         self.fix_keyring_permission()
         self.wait_for_ceph_health()
 
+    def run_haproxy(self):
+
+        """
+        task:
+            ceph-ansible:
+                haproxy: true
```
Comment (vasukulkarni) [inline teuthology/task/ceph_ansible.py]: can you add role definition that is needed like haproxy.i and haproxy.1

Comment (smanjara) [reply teuthology/task/ceph_ansible.py]: I have already added in 'groups to roles' dict and the deployment was successful. Is it required elsewhere?

Comment (smanjara) [reply teuthology/task/ceph_ansible.py]: Updated the doc string

**Comment by vasukulkarni** [comment]
Closing this in favor of #1232

**Pull request closed without merging**